translations/messages+intl-icu.de.yml

```diff
@@ -1012,6 +1026,8 @@ error.copy: "Leider ist beim Kopieren ein Fehler aufgetreten."
 error.copy.files: "Die Dateien der Stellungnahme {externId} konnten nicht kopiert werden."
 error.copy.mapfile: "Die Einzeichnung der Stellungnahme {externId} konnte nicht kopiert werden."
 error.changes.not.saved: "Beim Speichern der Änderungen ist ein Fehler aufgetreten."
+error.customer.contact.phone_or_email: "Bitte geben Sie entweder eine gültige E-Mail-Adresse oder Telefonnummer ein."
+error.customer.contact.title: "Bitte geben Sie Namen für den Kontakt ein."
```

Comment:
since this is not custom specific, I suggest to rename the keys to something like `error.phone_or_email` (here and above)

Reply:
Done with [3f83494](https://github.com/demos-europe/demosplan-core/pull/2139/commits/3f834942d1201ec22435b435febaa0b6eae40b45)